src/components/panel/panel.tsx

```diff
@@ -56,8 +56,14 @@ export class Panel implements ConditionalSlotComponent, InteractiveComponent {
   @Prop({ mutable: true, reflect: true }) closed = false;
 
   @Watch("dismissed")
+  dismissedHandler(value: boolean): void {
+    this.closed = value;
+    this.calcitePanelDismissedChange.emit();
+  }
+
   @Watch("closed")
-  dismissedHandler(): void {
+  closedHandler(value: boolean): void {
+    this.dismissed = value;
     this.calcitePanelDismissedChange.emit();
```

Comment:
Does this still need to be here? Can you make sure the event isnt being fired twice?

Reply:
It is firing twice. limited it to single emit now.

Reply:
Can we move this to the dismissed watcher?